main.go

```diff
@@ -64,9 +64,10 @@ func runServer() {
 
 	// kingpin splits on newline (?); split our tags on ',' instead
 	memberTags := util.SplitTags(*tags)
+	etcdMemberList := strings.Split(*etcdMembers, ",")
```

Comment:
I think you should use the util.SplitTags() because that will allow you to have spaces in the line.